gzlib.c

```diff
@@ -523,3 +523,13 @@ void Z_INTERNAL gz_error(gz_state *state, int err, const char *msg) {
     }
     (void)snprintf(state->msg, strlen(state->path) + strlen(msg) + 3, "%s%s%s", state->path, ": ", msg);
 }
+
+#ifdef ZLIB_COMPAT
+/* portably return maximum value for an int (when limits.h presumed not
+   available) -- we need to do this to cover cases where 2's complement not
+   used, since C standard permits 1's complement and sign-bit representations,
+   otherwise we could just use ((unsigned)-1) >> 1 */
```

Comment:
The rationale here doesn't match the code below...

Reply:
Yeah I can probably just remove the comment.

Reply:
Fixed.